src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -121,6 +121,12 @@ public static class ScriptConstants
 
         public const string AzureWebJobsHostsContainerName = "azure-webjobs-hosts";
 
+        public const string DefaultExtensionBundleDirectory = "FuncExtensionBundles";
+        public const string ExtensionBundleDirectory = "ExtensionBundles";
+        public const string ExtensionBundleCdnUri = "https://functionscdn.azureedge.net/public/extensionBundles";
+        public const string ExtensionBundleMetadatFile = "bundle.json";
+        public const string VersionIndexFile = "index.json";
```

Comment:
Follow same naming convention. Prefix with ExtensionBundle.
Change to ExtensionBundleDefaultDirectory, ExtensionBundleVersionIndexFile